CMakeLists.txt

```diff
@@ -7,9 +7,11 @@
 #        All rights reserved.
 #------------------------------------------------------------------------------#
 
-# Build notes: http://rtt/redmine/projects/draco/wiki/Configuration_and_Building
+# Build notes: 
+#   https://rtt.lanl.gov/redmine/projects/draco/wiki/
+#   https://rtt.lanl.gov/redmine/projects/draco/wiki/Common_Configure_Options
 
-cmake_minimum_required(VERSION 3.5.0 FATAL_ERROR)
+cmake_minimum_required(VERSION 3.9.0 FATAL_ERROR)
```

Comment:
It looks like the CI build is failing due to the version.
Is this why this is WIP?